test/entrypoint.sh

```diff
@@ -30,6 +30,13 @@ MYSQL_CONTAINER=1 $DIR/create_db.sh
 # Set up rabbitmq exchange
 rabbitmq-setup -server amqp://boulder-rabbitmq
 
+# Make sure SoftHSM container is up and using the correct key
+wait_tcp_port boulder-hsm 5657
+if [ ! -f test/test-ca.key.der ]; then
+    openssl rsa -in test/test-ca.key -inform pem -out test/test-ca.key.der -outform der
+fi
+PKCS11_PROXY_SOCKET="tcp://boulder-hsm:5657" pkcs11-tool --module=/usr/local/lib/libpkcs11-proxy.so --write-object test/test-ca.key.der --type privkey --label key_label --pin 5678 --login --so-pin 1234
```

Comment:
That would require rebuilding the `boulder-hsm` image every time we wanted to change/add keys which is what I was trying to avoid. This is an extremely cheap operation so I don't really see the downside (it also has the side effect of testing the HSM container is properly setup with the correct PINs etc before trying to spin up all the boulder services).